src/main/java/com/openshift/restclient/capability/resources/IPodExec.java

```diff
@@ -0,0 +1,150 @@
+/******************************************************************************* 
+ * Copyright (c) 2016 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package com.openshift.restclient.capability.resources;
```

Comment:
please repackage here: com.openshift.restclient.api.capabilities to align more with where resources are on the server